seqr/utils/middleware.py

```diff
@@ -33,10 +36,13 @@ class JsonErrorMiddleware(MiddlewareMixin):
     @staticmethod
     def process_exception(request, exception):
         if request.path.startswith('/api'):
-            exception_json = {'message': str(exception)}
+            try:
+                exception_json = {'message': str(exception)}
+            except Exception:
+                exception_json = {'message': unicode_str(exception)}
```

Comment:
I really dislike allowing python 2 strings here, especially since we will stop doing that when we switch to python 3. The idea behind this conversion is switch over to behaving in python 2 to the way it will in python 3, not to unnecessarily support python 2 behavior.  What I meant by using try-except here is we call `str(exception)` using the future `str` module, and if that throws an error we just return something generic like "An error occurred". Again, handling uncaught exceptions with unicode is an edge case I don't want us to worry about too much so I'd rather not write hard to maintain code to support it

Reply:
thinking about this more, why not use `str` with `ignore` set so it will just strip out unicodes?

Reply:
Updated